lib/utils/configurationComment.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const { assertString } = require('./validateTypes');
+
+const DISABLE_COMMAND = '-disable';
+const DISABLE_LINE_COMMAND = '-disable-line';
+const DISABLE_NEXT_LINE_COMMAND = '-disable-next-line';
+const ENABLE_COMMAND = '-enable';
+
+const ALL_COMMANDS = new Set([
+	DISABLE_COMMAND,
+	DISABLE_LINE_COMMAND,
+	DISABLE_NEXT_LINE_COMMAND,
+	ENABLE_COMMAND,
+]);
+
+const DEFAULT_COMMENT_PREFIX = 'stylelint';
```

Comment:
[suggest] For more accurate naming, how about the following?

```suggestion
const DEFAULT_COMMENT = 'stylelint';
```

or

```suggestion
const DEFAULT_CONFIGURATION_COMMENT = 'stylelint';
```

Reply:
Updated to `DEFAULT_CONFIGURATION_COMMENT`